packages/vc-handler-ld-local/src/suites/BbsBlsSignature2020.ts

```diff
@@ -0,0 +1,120 @@
+import { blsSign } from '@mattrglobal/node-bbs-signatures'
+import { Bls12381G2KeyPair, BbsBlsSignature2020 as MattrBbsBlsSignature2020 } from '@mattrglobal/jsonld-signatures-bbs'
+import { IKey, TKeyType, VerifiableCredential } from '@veramo/core'
+import { asArray } from '@veramo/utils'
+import suiteContext2020 from 'ed25519-signature-2020-context'
+import * as u8a from 'uint8arrays'
+
+import { SphereonLdSignature } from '../ld-suites'
+
+export class SphereonBbsBlsSignature2020 extends SphereonLdSignature {
+  constructor() {
+    super()
+    // Ensure it is loaded
+    suiteContext2020?.constants
+  }
+
+  getSupportedVerificationType(): string {
+    return 'Bls12381G2Key2020'
+  }
+
+  getSupportedVeramoKeyType(): TKeyType | 'Bls12381G2' {
+    return 'Bls12381G2'
```

Comment:
create a enum for this

Reply:
TKeyType is an union enum and already contains Bls12381G2. Fixed anyway

Reply:
the point is to have 1 place where the text is placed so that you do not get miss types

Reply:
Nah, that single point is actually this interface. Please stay as close to upstream as possible